### ISSUE TITLE
Show sound volume in decibels

### ISSUE THREAD

**Issue by matj1** [polybar/polybar#1886]
I'd like to see sound volume in the module `pulseaudio` in decibels but on the wiki page for the module `pulseaudio`, there is shown only that an available token for displaying volume is `%percentage%` and I don't know if that's all or there are others.

I can get the volume in decibels by parsing the output of `pactl list sinks` where is a line containing

    Hlasitost: front-left: 32967 /  50% / -17,90 dB,   front-right: 32967 /  50% / -17,90 dB

among many other lines and it's quite hard to parse.

I'd like to have a token like `%decibels%` in the label-volume format string that makes it show the volume in decibels.

**Comment by patrick96** [comment]
For anyone wanting to try their hands on this have a look at the following documentation,  the [`pa_sw_volume_to_dB`](https://freedesktop.org/software/pulseaudio/doxygen/volume_8h.html#a297851419c1e994d9fd2cfbb49aa480a) function and how we [get the volume in percent](https://github.com/polybar/polybar/blob/master/src/adapters/pulseaudio.cpp#L152)

https://www.freedesktop.org/wiki/Software/PulseAudio/Documentation/Developer/Clients/WritingVolumeControlUIs/

**Comment by patrick96** [comment]
Implemented in #1894
